### PR TITLE
Cumulus 313

### PR DESCRIPTION
**Pull request by laurenfrederick**

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 * [Introduction](README.md)
 * [Architecture](docs/architecture.md)
 * [Protocol](docs/protocol.md)
-* [Input & Ouptut](docs/input_output.md)
+* [Input & Output](docs/input_output.md)
```
Comment (bilts) [inline SUMMARY.md]: "Ouptut" was growing on me.  Very Ancient Egypt.

**Review by bilts — APPROVED** [review]
Looks great. Thought I'd have more than just the snarky comment, but I don't see any changes to be made.  Thanks!

**Review by scisco — COMMENTED** [review]
I only have one comment

File: docs/input_output.md [hunk 2]
```diff
@@ -90,28 +25,13 @@ Output of the task application code:
...
-
-When a workflow receives a large message, the Cumulus Message Adapter is responsible for fetching the JSON message document and passing it to the task.  When a task output message is too big, the Cumulus Message Adapter will store the message to S3 Bucket under `$.cumulus_meta.buckets.internal`, and return a new message with an S3 reference as in the above example.
-
-#### 2. Resolve URL templates in the task configuration
+#### 1. Resolve URL templates in the task configuration
 
 In the workflow configuration, each task has its own configuration, and it can use URL template as a value to achieve simplicity or for values only available at execution time. The Cumulus Message Adapter resolves the URL templates and then passes message to next task. For example, given a task which has the following configuration:
 
     Discovery:
         config:
```
[the remark this answers is not in the document]
Comment (laurenfrederick) [reply docs/input_output.md]: Updated

**Review by scisco — APPROVED** [review]

**Pull request merged**